src/index.ts

```diff
@@ -208,7 +252,24 @@ export class CheckoutInstance {
   }
 
   private mapOptionsToManifest(manifest: CheckoutManifest, options: CheckoutOptions) {
-    const { hideDetails, language, logoUrl, display, backgroundColor } = options
+    const {
```

Comment:
Is this just to avoid repeating `options.` below in each property?

Reply:
Yes probably. I stood with the previous code.